### ISSUE TITLE
Finetuning

### ISSUE THREAD

**Issue by abhimanyuiris96** [rishigami/Swin-Transformer-TF#17]
Has anyone tried fine-tuning the pre-trained networks to their own datasets?
Mine gets stuck at 20% accuracy and doesn't move from there.

**Comment by almahdir1** [comment]
I am wondering how I can fine tune on different dataset with different input size? Any help would highly appreciated.
